src/inmanta/db/versions/v202310090.py

```diff
@@ -0,0 +1,35 @@
+"""
+    Copyright 2023 Inmanta
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+    Contact: code@inmanta.com
+"""
+from asyncpg import Connection
+
+
+def convert_setting_to_str(setting: str) -> str:
+    return f"""
+    UPDATE environment
+    SET settings=jsonb_set(settings, '{{{setting}}}'::TEXT[], to_jsonb(settings->>'{setting}'), FALSE)
+    WHERE settings ? '{setting}'
+    """
```

Comment:
Let's add a comment here saying that the `->>` operator returns something of type TEXT, so that it's clear where in the query the type conversion actually happens.